client/app/components/onboardingGuide/GuideCoverPhotoPage.jsx

```diff
@@ -0,0 +1,62 @@
+import React, { PropTypes } from 'react';
+import scss from './styles.scss';
+
+const GuideCoverPhotoPage = (props) => {
+  const handleClick = function handleClick(e, path) {
+    e.preventDefault();
+    props.changePage(path);
+  };
```

Comment:
This here is probably a good candidate to move to HOC or even better, it's own component